Homework10/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Homework10
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var woman = new GenericArray<Woman>(2);
+            var man = new GenericArray<Man>(1);
+
+            GenerateElements(2, woman);
+            GenericArray<Woman>.ToString(woman);
+
+            GenerateElements(1, man);
+            GenericArray<Man>.ToString(man);
+        }
+
+        public static void GenerateElements<T>(int numberOfElements, GenericArray<T> generatedElements) where T : Human, new()
+        {
+            string FName;
```

Comment:
Naming rule - local variables for methods starts with lower case letter

Reply:
oh, yep